Tools/build/generate_sbom.py

```diff
@@ -20,6 +21,12 @@
     "CC0-1.0",
     "Apache-2.0",
     "BSD-2-Clause",
+    "BSD-3-Clause",
```

Comment:
These are the new license expressions of the subdependencies.

Reply:
Pop them all in alphabetical order? Makes it easier to scan to check if ones there, especially when the list grows.